.github/workflows/deployment.yml

```diff
@@ -0,0 +1,79 @@
+name: safe-gelato-relay-service
+
+on: [ push, pull_request ]
+
+jobs:
+
+  docker-publish-staging:
```

Comment:
We have deployment jobs in both `.github/workflows/ci.yml` and `.github/workflows/deployment.yml` that target the same environment.

Why is that? 🤔

Reply:
removed deployment.yml